packages/scripts/package.json

```diff
@@ -39,30 +39,37 @@
 		"@wordpress/eslint-plugin": "file:../eslint-plugin",
 		"@wordpress/jest-preset-default": "file:../jest-preset-default",
 		"@wordpress/npm-package-json-lint-config": "file:../npm-package-json-lint-config",
+		"@wordpress/postcss-plugins-preset": "file:../postcss-plugins-preset",
 		"@wordpress/prettier-config": "file:../prettier-config",
 		"babel-jest": "^25.3.0",
 		"babel-loader": "^8.1.0",
 		"chalk": "^4.0.0",
 		"check-node-version": "^3.1.1",
 		"command-exists": "^1.2.8",
 		"cross-spawn": "^5.1.0",
+		"css-loader": "^3.5.2",
 		"decompress-zip": "^0.2.2",
 		"dir-glob": "^3.0.1",
 		"eslint": "^6.8.0",
 		"eslint-plugin-markdown": "^1.0.2",
 		"got": "^10.7.0",
+		"ignore-emit-webpack-plugin": "^2.0.2",
 		"jest": "^25.3.0",
 		"jest-puppeteer": "^4.4.0",
 		"js-yaml": "^3.13.1",
 		"lodash": "^4.17.15",
 		"markdownlint": "^0.18.0",
 		"markdownlint-cli": "^0.21.0",
+		"mini-css-extract-plugin": "^0.9.0",
 		"minimist": "^1.2.0",
+		"node-sass": "^4.13.1",
```

Comment:
I hope we don't come to regret foisting this troublesome dependency upon consumers of `@wordpress/scripts` 😬

Reply:
It is what it is, we use it in Gutenberg as well.

Reply:
Maybe separately, it's something to consider to the general problem of the amount (and difficulty) of the specific dependencies we ship here. Whether it's something we can defer with some sort of [lazy importing](https://www.npmjs.com/package/import-lazy), or splitting off to more granular packages, or some sort of peer dependency pattern to leverage the features the developer opts in to.

I just worry that `@wordpress/scripts` tries to do everything and the pattern to ship it all in the same package this way won't doesn't scale well to accommodate.

Reply:
Though, to be clear, my issue with SASS dependency specifically is less of size, and more of how it's built from source, and all the usability problems that stem from this (e.g. [Node version correspondence](https://www.npmjs.com/package/node-sass)).

Reply:
I would suggest changing this to Dart SASS (`sass` package). It's the [primary implementation](https://sass-lang.com/dart-sass) for SASS, and saves contributors all the native compilation trouble `node-sass` brings. `sass-loader` picks it up without any additional configuration.

Reply:
@yscik and @aduth – I filed https://github.com/WordPress/gutenberg/issues/22729 to continue the discussion. I don't mind switching. In this PR, the goal was to mirror as much as possible from what Gutenberg uses – for better or worse :)